cookiecutter.json

```diff
@@ -20,4 +20,4 @@
     
     "include_ReadTheDocs": ["y", "n"],
   "_cms_cc_version": 1.5
```

Comment:
Is this the release version of this repository? Should it be incremented for a change like this?

Reply:
That is the way we can track what version of the cookiecutter this is, it should be bumped for things like this. There is no good way to version cookiecutters in such a way that the outputs are stamped unless you hardcode it somewhere, and having it in the cookiecutter JSON file where the user would not see it made the most sense. Admittedly, this relies on undocumented and possibly unintended behavior from the Cookiecutter program where options with a leading `_` are ignored but carried through as an accessible attribute. I've [raised the issue about how to formally track cookiecutter versions](https://github.com/cookiecutter/cookiecutter/issues/1135), but its never received a reply.